### PR TITLE
Disabled link at current page

### PR DESCRIPTION
**Pull request by aquy**
linkpager disabled link at current page

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | no

### CODE REVIEW

**Comment by samdark** [comment]
Why?

**Comment by aquy** [comment]
link page to itself on the site bad for SEO and the user is already on this page

**Comment by samdark** [comment]
OK. Makes sense.

**Comment by bashkarev** [comment]
Google and others, remove the current link.
Maybe replace `a` to `span`?
And to make it optional.

**Comment by samdark** [comment]
Yes. That is a good idea.

**Comment by aquy** [comment]
I've added the option to replace `a` to `span` - `disabledCurrentPageLink`

File: framework/widgets/LinkPager.php [hunk 1]
```diff
@@ -124,6 +124,10 @@ class LinkPager extends Widget
      * @var bool Hide widget when only one page exist.
      */
     public $hideOnSinglePage = true;
+    /**
+     * @var bool replace a to span the current page
+     */
+    public $disabledCurrentPageLink = false;
```
Comment (samdark) [inline framework/widgets/LinkPager.php]: `disabledCurrentPageLink` → `disableCurrentPageLink`.

**Review by samdark — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Review by SilverFire — CHANGES_REQUESTED** [review]
Could you update [tests](https://github.com/yiisoft/yii2/blob/master/tests/framework/widgets/LinkPagerTest.php) to reflect the changes?

**Comment by aquy** [comment]
Yes, tests have been updated

**Comment by thyseus** [comment]
How about making this to be true by default?

**Comment by samdark** [comment]
I think it's OK to make it true by default.

**Comment by bashkarev** [comment]
This can break the page layout

**Comment by samdark** [comment]
How?

**Comment by bashkarev** [comment]
if not use bootstrap
```html
<ul>
    <li><a href="/?page=1">url</a></li>
    <li><span>current</span></li>
</ul>
```
```css
ul > li > a {
    float: left;
    display: block; // etc
}
```

**Comment by samdark** [comment]
Right. Then it's not a good idea :)

**Review by SilverFire — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request closed without merging**